schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+	id INT GENERATED BY DEFAULT AS IDENTITY,
```

Comment:
- [ ] (optional). It would improve the structure of this table if you made the `id` the primary key